styleguide/app/index.html

```diff
@@ -19,63 +21,72 @@
-                <li>
-                    <a href="http://maps.coastalresilience.org/pugetsound" target="_blank">Washington</a>
+                <li class="divider"></li>
+                <li></li>
+                <li class="dropdown-submenu">
+                    <a id="region-header-menu" href="#">
+                        Other Regions
+                        <i class="dropdown-icon icon-angle-right"></i>
+                    </a>
+                    <ul class="dropdown-menu">
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/california/">California <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/connecticut">Connecticut <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/global/">Global <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/gsvg/">Grenada, St. Vincent &amp; the Grenadines <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/mar/">MesoAmerican Reef <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/newyork">New York <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/newjersey">New Jersey <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/seflorida/">Southeast Florida <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/unitedstates/">United States <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/usvi/">U.S. Virgin Islands <i class="icon-link-ext"></i></a>
+                        </li>
+                        <li>
+                            <a target="_blank" href="http://maps.coastalresilience.org/pugetsound">Washington <i class="icon-link-ext"></i></a>
+                        </li>
+                    </ul>
                 </li>
             </ul>
         </div>
+        <div class="nav-region-subtitle">
+            <a href="/snippet_index.html">View "code snippet" style guide</a>
```

Comment:
Consider making this `color: #fff` so that it contrasts a bit more.
![screenshot from 2017-02-21 17 42 23](https://cloud.githubusercontent.com/assets/1014341/23188553/208ea0d6-f85d-11e6-8c78-6f7f200c87be.png)